### PR TITLE
UI changes to manager selection

### PR DESCRIPTION
**Pull request by harish86**

### CODE REVIEW

**Comment by antoineherzog** [comment]
@harish86 can we remove the member of the card ?

**Comment by antoineherzog** [comment]
@harish86 you need to solve conflict also.

File: web/static/components/cards/select-member.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    },
+    selectMember: function(user) {
+      if(this.value !== user.id) {
+        this.$emit('input', user.id);
+        this.$emit('change');
+      }
+      this.editMode = false;
+    },
+    cancel: function() {
+      this.v = this.value;
```
Comment (parallel588) [inline web/static/components/cards/select-member.vue]: Hi @harish86, I don't sure but I don't see where we use this.v.

Comment (harish86) [inline web/static/components/cards/select-member.vue]: @parallel588 I removed it.

**Pull request merged**